FeedAPIChallenge/Feed API/FeedImagesMapper.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  FeedImageMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Hardeep Singh on 15/09/21.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+final class FeedImagesMapper {
+	private struct Root: Codable {
+		let items: [Item]
+		var images: [FeedImage] {
+			items.compactMap { $0.image }
+		}
+	}
+
+	private struct Item: Codable {
+		let image_id: UUID
+		let image_url: String
+		let image_loc: String?
+		let image_desc: String?
+
+		var image: FeedImage? {
+			guard let imageURL = URL(string: image_url) else {
+				return nil
+			}
+			return FeedImage(id: image_id,
+			                 description: image_desc,
+			                 location: image_loc,
+			                 url: imageURL)
+		}
+	}
+
+	private static var OK_200: Int { 200 }
```

Comment:
It is absolutely ok to use implicit returns but they are not used in this project. It is important to be consistent and keep nice and concise code style. Please make the return explicit
```suggestion
	private static var OK_200: Int { return 200 }
```